tools/bridge/tests/test_confirmation_sender.py

```diff
@@ -194,3 +195,24 @@ def test_pending_transfers_are_cleared(
         assert confirmation_sender.pending_transaction_queue.qsize() == 0
     finally:
         greenlet.kill()
+
+
+def test_do_not_confirm_as_non_bridge_validator(
+    confirmation_sender_with_non_validator_account, transfer_queue, transfer_event
+):
+    try:
+        greenlet = gevent.spawn(confirmation_sender_with_non_validator_account.run)
+
+        assert (
+            confirmation_sender_with_non_validator_account.pending_transaction_queue.empty()
+        )
+
+        transfer_queue.put(transfer_event)
+        gevent.sleep(0.1)
```

Comment:
Just a comment: I'm not too happy about this kind of test as there might be plenty of other reasons why the queue is still empty after the `gevent.sleep` (e.g. if there's some kind of RPC request if we run the test on a live chain or if there's some other `sleep` in there for whatever reason). I don't know a better way to do it though.

Reply:
Yeah, I also thought about it while taking the template from you. But as you I can't come up with a better solution for now. I would add a TODO comment on it, just that it is not totally forgotten. Especially when we do some live chain tests and this case fails, the developer see this comment and get an hint.

Reply:
:heavy_check_mark: